jobserver/templates/workspace_log.html

```diff
@@ -25,6 +25,11 @@ <h2>{{ workspace.name }}</h2>
 
   </div>
 
+  <p class="mb-4">
+    Below are the logs for each job run in this workspace.  They are grouped
```

Comment:
..for each job run in the {{ workspace.name }} [link] workspace of the project {{ project.name }} [link]